File: Level 1/로또의 최고 순위와 최저 순위/Solution.java
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] answer = new int[2];
        int cnt = 0;
        int zero_cnt = 0;
        for(int i=0; i<lottos.length; i++) {
            if(lottos[i] == 0) zero_cnt++;
            for(int j=0; j<win_nums.length; j++) {
                if(lottos[i] == win_nums[j]) cnt++;
            }
        }
        if(cnt + zero_cnt >= 2) answer[0] = 7-(cnt + zero_cnt);
        else answer[0] = 6;
        if(cnt >=2) answer[1] = 7-cnt;
        else answer[1] = 6;
        return answer;
    }
}

File: Level 1/로또의 최고 순위와 최저 순위/solution.js
function solution(lottos, win_nums) {
    var answer = [];
    let count = 0;
    let count_zero = 0;
    for(let i of lottos) {
        if(i == 0) {
            count_zero++;
            continue;
        }
        if(win_nums.indexOf(i) != -1) count++;
    }
    let max = count + count_zero;
    answer[0] = (max >= 2) ? 7 - max : 6;
    answer[1] = (count >= 2) ? 7 - count : 6;
    return answer;
}